### ISSUE TITLE
GraphicsMagick 1.3.28 is removed from sourceforge.net

### ISSUE THREAD

**Issue by bormant** [Ray-V/tde-slackbuilds#20]
https://github.com/Ray-V/tde-slackbuilds/blob/c18d07c7f04861ea5fd5a3d3424ab3b6012af609/Misc/GraphicsMagick/GraphicsMagick.SlackBuild#L27

1.3.28 is removed from souceforge.net.

Options are:
- bump version to 1.3.34
- use another mirror for 1.3.28, for ex.
http://ftp.icm.edu.pl/pub/unix/graphics/GraphicsMagick/1.3/
ftp://ftp.graphicsmagick.org/pub/GraphicsMagick/1.3/

**Comment by Ray-V** [comment]
I'm working on a general update and that is part of it - v1.3.34 works.  
I'll use one of the mirrors you've suggested, though, so that any future version changes won't break the build before the SlackBuild script has been updated.

The 14.0.7 builds are looking ok, but I haven't tested a cgit build yet so it'll be a few days before I push the changes.

In case you find any more issues with the current repo, I've attached the [draft] diff to show what'll be covered.

[20191227-master.diffs.txt](https://github.com/Ray-V/tde-slackbuilds/files/4028161/20191227-master.diffs.txt)

---------------
If you've got the time or inclination, one of the things I've been looking at is the tdeedu build.  
Python scripting shows as an option in Kig with an autotools build, but not with a cmake build.  

If I replace libkigpart.so in the cmake build with that from the autotools build, the python scripting option then shows and works.  
I initially thought that it might be that the cmake build isn't linking against libboost_python - I tried adding that, but that didn't work, at least not the way I did it ...  
I'll be interested in any feedback .. whether it's a build issue or needs to be raised upstream.
